### PR TITLE
Stock/History pages

### LINKED ISSUE #31: Save Stock history to local file
> At any time, the user should be able download their purchase and sales history into a local file (any reasonable format is fine) on their own machines.

### PR DESCRIPTION
**Pull request by rayrobdod**
Closes #31, at least as well as can be done without #25 or #18

### CODE REVIEW

File: src/SE344/Views/Stocks/History.cshtml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                        <td>@t.Value.NumShares</td>
+                        <td>@t.Value.TotalPrice</td>
+                    </tr>
+                }
+            </tbody>
+        </table>
+    </div>
+
+    <div id="historyActions" class="btn-group btn-group-vertical col-md-2">
+        <!-- TODO: horizontal alignment on tall, skinny screens -->
```
Comment (mikairi) [inline src/SE344/Views/Stocks/History.cshtml:46]: Can you open a new issue for this? Could be tagged as "Enhancement". Just so we don't lose track of these TODO comments later.

**Comment by rayrobdod** [comment]
Again, I wasn't inclined to make issues about bugs until after code was merged to master.

File: src/SE344/Views/Stocks/History.cshtml (hunk 1 above)
Comment (rayrobdod) [inline src/SE344/Views/Stocks/History.cshtml:46]: Again, I wasn't inclined to make issues about bugs until after code was merged to master.

Comment (mikairi) [inline src/SE344/Views/Stocks/History.cshtml:46]: I didn't say it was a :bug: though, which is why I mentioned "Enhancement".

I also didn't mean you had to do it right away. It was more of a reminder so feel free to wait until after this is merged to create the issue.

**Comment by mikairi** [comment]
:+1:

**Pull request merged**